apps/mobile/src/components/widgets/collectibles/collectibles-header.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+
+import { t } from '@lingui/macro';
+
+import { Box, ChevronRightIcon, Chip, SheetRef, Text } from '@leather.io/ui/native';
+
+import { FiatBalance } from '../components/balance/fiat-balance';
+
+interface CollectiblesHeaderProps {
+  collectibleCount: number;
+  sheetRef: React.RefObject<SheetRef>;
+  totalBalance: string;
+}
+
+function CollectiblesHeaderText() {
+  return <Text variant="heading05">{t`My collectibles`}</Text>;
+}
+
+export function CollectiblesHeader({ collectibleCount, totalBalance }: CollectiblesHeaderProps) {
```

Comment:
All the widgets have this same header so I will refactor this to keep things DRY in the next PR